src/libraries/System.Text.Json/gen/JsonSourceGenerator.Emitter.cs

```diff
@@ -1009,7 +1007,9 @@ private static void GenerateTypeInfoFactoryHeader(SourceWriter writer, TypeGener
                     /// <summary>
                     /// Defines the source generated JSON serialization contract metadata for a given type.
                     /// </summary>
+                    #nullable disable annotations // Marking the property type as nullable-oblivious.
                     public {{typeInfoFQN}} {{typeInfoPropertyName}}
+                    #nullable enable annotations
```

Comment:
Are there other places where the generic type arguments used end up surfacing and that would also benefit from being made oblivious?

Reply:
That's the only public API surfacing generic metadata types. Everything else is using the non-generic `Type => JsonTypeInfo` resolver methods.